WordPress/Classes/ViewRelated/Me/My Domains/MyDomainsViewController.swift

```diff
@@ -0,0 +1,191 @@
+        super.viewDidLoad()
+        title = Strings.title
+
+        WPStyleGuide.configureColors(view: view, tableView: nil)
+        navigationItem.rightBarButtonItem = .init(systemItem: .add)
+        let addToSiteAction = menuAction(withTitle: Action.addToSite.title) { action in
+            print("add to site")
+        }
+        let purchaseAction = menuAction(withTitle: Action.purchase.title) { action in
+            print("purchase")
+        }
+        let transferAction = menuAction(withTitle: Action.transfer.title) { action in
+            print("transfer")
+        }
+
+        let actions = [addToSiteAction, purchaseAction, transferAction]
+        let menu = UIMenu(title: "", image: nil, identifier: nil, options: .displayInline, children: actions)
+        navigationItem.rightBarButtonItem?.menu = menu
+
+        self.setupSubviews()
+        self.observeState()
+        self.viewModel.loadData()
+    }
+
+    private func setupSubviews() {
+        // Setup search bar
+        let searchController = UISearchController(searchResultsController: nil)
+        searchController.delegate = self
+        searchController.searchBar.delegate = self
+        searchController.searchBar.placeholder = Strings.searchBar
+        self.navigationItem.searchController = searchController
+
+        // Setup tableView
+        self.tableView.translatesAutoresizingMaskIntoConstraints = false
+        self.tableView.dataSource = self
+        self.tableView.delegate = self
+        self.tableView.contentInset.top = Layout.interRowSpacing
+        self.tableView.sectionHeaderHeight = Layout.interRowSpacing / 2
+        self.tableView.sectionFooterHeight = tableView.sectionHeaderHeight
+        self.tableView.verticalScrollIndicatorInsets.top = -tableView.contentInset.top
+        self.tableView.register(MyDomainsTableViewCell.self, forCellReuseIdentifier: CellIdentifiers.myDomain)
+        self.tableView.register(MyDomainsActivityIndicatorTableViewCell.self, forCellReuseIdentifier: CellIdentifiers.activityIndicator)
+        self.view.addSubview(tableView)
+        self.view.pinSubviewToAllEdges(tableView)
+    }
+
+    private func observeState() {
+        self.viewModel.$state.sink { [weak self] state in
+            guard let self else {
+                return
+            }
+            self.state = state
+            switch state {
+            case .normal, .loading:
+                self.tableView.reloadData()
+            case .error:
+                break
+            case .empty:
+                break
+            }
+        }.store(in: &cancellable)
+    }
+
+    private func menuAction(withTitle title: String, handler: UIActionHandler) -> UIAction {
+        .init(
+            title: title,
+            image: nil,
+            identifier: nil,
+            discoverabilityTitle: nil,
+            attributes: .init(),
+            state: .off,
+            handler: { (action) in
+            // Perform action
+        })
+    }
```

Comment:
Same as this [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/21860/files#r1369490185). This code is related to the "+" menu context, and it will be removed.